docs/advanced-topics/serializer.md

```diff
@@ -1,6 +1,9 @@
 Datatype and Attribute Serializer Configuration
 ===============================================
 
+!!! Info
+    Datatype and Attribute Serializer Configuration are added in JanusGraph version `0.3.2`
```

Comment:
No, serialization was already implemented. I mean only `ObjectNode` and `ArrayNode`.
I.e. 
`ObjectNode` and `ArrayNode` serializers are added in JanusGraph version `0.3.2`